presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -322,6 +322,15 @@ public String getEnvironmentName()
             return name;
         }
 
+        public Builder containerDependsOn(String container, String otherContainer)
+        {
+            checkState(containers.containsKey(container), "Container with name %s is not registered", name);
```

Comment:
Also instead we could maybe add `getContainer` method to builder and go with:
```java
        builder.addContainers(createZookeeper(), createKafka())
                .configureContainer("kafka", kafkaContainer -> kafkaContainer.dependsOn(builder.getContainer("zookeeper")));
```

But I think helper method is nice

Reply:
That was my initial thought. I dont't know if builder should have get method though (what's your take on that?)

Reply:
I would not object if there are more cases when that would be useful.
But the helper method you added is fine too. And it does `checkState` calls which would make the `configureContainer` bloated.